src/posts/faq.md

```diff
@@ -18,15 +18,12 @@ AI safety researchers (who are the experts on this topic) are divided on this qu
 Note that no (surveyed) AI safety researchers believe that there's a 0% chance.
 However, there might be selection bias here: people who work in the AI safety field are likely to do so because they believe preventing bad AI outcomes is important.
 
-If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/).
+If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/), with a median of 5%.
 A small minority, about 4% of them, believe that the alignment problem is not a real problem.
 Note that there might be a selection bias here in the opposite direction: people who work in AI are likely to do so because they believe AI will be beneficial.
 
 _Imagine you're invited to take a test flight on a new airplane_.
-The airplane safety experts on average think there's a 30% chance it would crash.
```

Comment:
Why remove this line? It makes the analogy more powerful and it's backed up by data.

Reply:
The number does not reflect how many people the risk is real, but how large the risk is.

AI safety researchers are the domain experts here.

If you want to know how many people don't believe the alignment problem is real, that one is included too. That would be analogous to saying "the earth is flat"

Reply:
AI safety researchers are the actual experts here, so I don't think the analogy is unfair. If experts are _this_ worried about it, we should be worried, too.